### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by johscheuer**
# Description

Fixes: https://github.com/FoundationDB/fdb-kubernetes-operator/issues/802

*Please include a summary of the change and which issue is addressed. If this change resolves an issue, please include the issue number in the description.*

## Type of change

*Please select one of the options below.*

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation
- Other

# Discussion

*Are there any design details that you would like to discuss further?*

# Testing

*Please describe the tests that you ran to verify your changes. Unit tests?
Manual testing?*

*Do we need to perform additional testing once this is merged, or perform in a larger testing environment?*

# Documentation

*Did you update relevant documentation within this repository?*

*If this change is adding new functionality, do we need to describe it in our user manual?*

*If this change is adding or removing subreconcilers, have we updated the core technical design doc to reflect that?*

*If this change is adding new safety checks or new potential failure modes, have we documented and how to debug potential issues?*

# Follow-up

*Are there any follow-up issues that we should pursue in the future?*

*Does this introduce new defaults that we should re-evaluate in the future?*

### CODE REVIEW

**Comment by johscheuer** [comment]
Fixed and related to https://github.com/helm/kind-action/issues/42 the fix was to upgrade to 0.11.1 (kind)

File: scripts/setup_kind_local_registry.sh [hunk 1]
```diff
@@ -29,16 +29,3 @@ EOF
 # connect the registry to the cluster network
 # (the network may already be connected)
 docker network connect "kind" "${reg_name}" || true
-
-# Document the local registry
-cat <<EOF | kubectl apply -f -
```
Comment (johscheuer) [inline scripts/setup_kind_local_registry.sh:34]: We actually don't need this

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -68,9 +59,12 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         # With a new supported Kubernetes version we should adjust the version
-        kubever: [ "v1.17.0", "v1.18.0", "v1.19.0", "v1.20.0" ]
+        # See https://kubernetes.io/releases for the current releases
+        kubever: [ "v1.19.0", "v1.20.0", "v1.21.1" ]
+        fdbver: ["6.2.30", "6.3.15"]
```
Comment (johscheuer) [inline .github/workflows/pull_request.yml:67]: I think it makes sense to test at least these version (we could also test `6.1` but I think `6.2` and `6.3` is fine.

**Review by 09harsh — APPROVED** [review]
Lgtm 👍

**Pull request merged**